scripts/Dockerfile

```diff
@@ -11,12 +11,197 @@ ENV DEBIAN_FRONTEND noninteractive
+	&& apt-get clean autoclean \
+	&& rm -rf /var/lib/apt/lists/{apt,dpkg,cache,log} /tmp/* /var/tmp/*
+
+ENV ROS_DISTRO melodic
+
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654 \
+	&& sh -c 'echo "deb http://packages.ros.org/ros/ubuntu `lsb_release -sc` main" > /etc/apt/sources.list.d/ros-latest.list' \
+	&& sh -c 'echo "deb http://packages.ros.org/ros-shadow-fixed/ubuntu `lsb_release -sc` main" > /etc/apt/sources.list.d/ros-shadow.list' \
+	&& apt-get update \
+	&& apt-get -y --quiet --no-install-recommends install \
+		geographiclib-tools \
+		libeigen3-dev \
+		libgeographic-dev \
+		libopencv-dev \
+		libyaml-cpp-dev \
+		python-pip \
+		python-tk \
+		ros-$ROS_DISTRO-gazebo-ros-pkgs \
+		ros-$ROS_DISTRO-jderobot-drones \
+		ros-$ROS_DISTRO-mav-msgs \
+		ros-$ROS_DISTRO-mavlink \
+		ros-$ROS_DISTRO-mavros \
+		ros-$ROS_DISTRO-mavros-extras \
+		ros-$ROS_DISTRO-octomap \
+		ros-$ROS_DISTRO-octomap-msgs \
+		ros-$ROS_DISTRO-pcl-conversions \
+		ros-$ROS_DISTRO-pcl-msgs \
+		ros-$ROS_DISTRO-pcl-ros \
+		ros-$ROS_DISTRO-ros-base \
+		ros-$ROS_DISTRO-rostest \
+		ros-$ROS_DISTRO-rosunit \
+		ros-$ROS_DISTRO-xacro \
+		xvfb \
+	&& geographiclib-get-geoids egm96-5 \
+	&& apt-get -y autoremove \
+	&& apt-get clean autoclean \
+	&& rm -rf /var/lib/apt/lists/{apt,dpkg,cache,log} /tmp/* /var/tmp/*
+
+# Install everything again for Python 2 because we could not get Firmware
+# to compile using catkin without it.
+RUN pip install wheel setuptools
+RUN pip install argcomplete argparse catkin_pkg catkin-tools cerberus coverage \
+    empy jinja2 matplotlib==2.2.* numpy pkgconfig px4tools pygments pymavlink \
+    packaging pyros-genmsg pyulog pyyaml requests rosdep rospkg serial six toml
+
+# bootstrap rosdep
+# RUN rosdep init && rosdep update
+RUN rosdep update
+
+## PX4 src
+RUN apt upgrade -y --quiet --no-install-recommends libignition-math2
+RUN git clone --recursive https://github.com/PX4/Firmware.git -b v1.11.3 \
+	&& cd /Firmware \
+	&& DONT_RUN=1 make px4_sitl_default gazebo
 
 # Get the Custom Robot Repository
 # and set the environment variables
-RUN mkdir -p /opt/jderobot && cd /opt/jderobot && git clone https://github.com/JdeRobot/CustomRobots.git && cd CustomRobots && git checkout melodic-devel && git pull origin melodic-devel
-ENV GAZEBO_MODEL_PATH=/opt/jderobot/CustomRobots/f1:/opt/jderobot/CustomRobots/roomba_robot
+RUN mkdir -p /opt/jderobot && cd /opt/jderobot && git clone https://github.com/JdeRobot/CustomRobots.git \
+    && cd CustomRobots && git checkout melodic-devel && git pull origin melodic-devel
+
+## Setting env vars
+ENV GAZEBO_RESOURCE_PATH "$GAZEBO_RESOURCE_PATH:/RoboticsAcademy/exercises/follow_line/web-template/launch"
+ENV GAZEBO_PLUGIN_PATH "$GAZEBO_PLUGIN_PATH:/opt/jderobot/CustomRobots/f1:/opt/jderobot/CustomRobots/roomba_robot:/Firmware/build/px4_sitl_default/build_gazebo"
+ENV GAZEBO_MODEL_PATH "$GAZEBO_MODEL_PATH:/Firmware/Tools/sitl_gazebo/models"
```

Comment:
Slight correction

`GAZEBO_MODEL_PATH` should contain `/opt/jderobot/CustomRobots/f1:/opt/jderobot/CustomRobots/roomba_robot`. By mistake, these directories are contained in `GAZEBO_PLUGIN_PATH`

Reply:
Done! Thanks for noticing!